users/views.py

```diff
@@ -86,124 +90,256 @@ def update_user(request):
+
+        if not month:
+            month = int(datetime.now().month)
+
+        action = 1 if next else -1
+
+        if month + action > 12:
+            year += 1
+            month -= 11
+        elif month + action < 1:
+            year -= 1
+            month += 11
         else:
-            events_count[event.date_time_start.date()] += 1
-
-    return max(events_count.values())
-
-
-def get_table_measurements(events):
-    """ get each tab in the table a measurement for the positioning
-
-        maximum row height, tab_top_positioning, event_margin_top, event_maring_bottom
-    """
-
-    max_amount_of_events = get_max_day_events(events)  # get max events in one day
-
-    tab_top_positioning = max_amount_of_events * 2 if max_amount_of_events > 2 else 4
-    maximum_row_height = max_amount_of_events * 5 if max_amount_of_events > 2 else 13
-
-    maximum_row_height = min(maximum_row_height, 30)
-    tab_top_positioning = min(tab_top_positioning, 13)
-
-    event_margin_top = min(tab_top_positioning, 13) - 4
-    event_maring_bottom = event_margin_top + 0.5
-
-    return (maximum_row_height, tab_top_positioning, event_margin_top, event_maring_bottom)
-
-
-def get_current_month_name(month=None):
-    """ get the current month name.
-
-        if month is given then get the name of that month
-        otherwise get this month name
-    """
-    current_date = datetime.now()
-    if month:
-        current_date = datetime(current_date.year, month, current_date.day)
-    return current_date.strftime("%B")
-
-
-def get_calendar_days(year=None, month=None):
-    """ return a matrix of dates. Each sublist is a week in the month, each week has 7 dates.
-        This list will include dates from the month before if the current month hasn't started with sunday """
-
-    current_week = list()
-    calendar_dates = list()
-    current_date = datetime.now()
-    if not year:
-        year = current_date.year
-    if not month:
-        month = current_date.month
-
-    # iterate over the list of all the current month dates.
-    # divide the list to each week and return the resulting matrix
-    for index, date_index in enumerate(cal.itermonthdays3(year, month)):
-        current_week.append(date_index)
-        if (index + 1) != 1 and (index + 1) % 7 == 0:
-            calendar_dates.append(current_week)
-            current_week = list()
-    calendar_dates.append(current_week)
-    return calendar_dates
+            month += action
+
+        return f"{year}/{month}"
+
+    @staticmethod
+    def get_max_day_events(events):
+        """ return the maximum events count in one day over all the events given """
+
+        events_count = {}
+        if len(events) == 0:
+            return 0
+
+        for event in events:
+            if not events_count.get(event.date_time_start.date()):
+                events_count[event.date_time_start.date()] = 1
+            else:
+                events_count[event.date_time_start.date()] += 1
```

Comment:
when you use "event.date_time_start.date()" 3 times 
I think it could be more readable to define it before and use it later like:
```suggestion
            event_start_date = event.date_time_start.date()
            if not events_count.get(event_start_date):
                events_count[event_start_date] = 1
            else:
                events_count[event_start_date] += 1
```

Reply:
Done. thanks 😄